server/includes/functions.js

```diff
@@ -65,12 +63,32 @@ export const checkInputLength = (requestObject, inputField) => {
   const error = {};
   if (typeof requestObject === 'object' && Array.isArray(inputField)) {
     for (counter = 0; counter < inputField.length; counter += 1) {
-      if (requestObject[inputField[counter]].length > 255) {
+      if (requestObject[inputField[counter]].trim().length > 255) {
         error[inputField[counter]] = 'Maximum character length exceeded';
+      } else if (requestObject[inputField[counter]].trim().length < 1) {
+        error[inputField[counter]] = 'Field cannot be empty';
       }
     }
   } else {
     return 'bad input to function';
   }
   return error;
 };
+
+/**
+ *
+ *
+ * @param {string} password
+ * @returns {string} -returns hashed password
+ */
+export const encryptPassword = (password) => {
+  let returnValue;
```

Comment:
I would suggest you try to use variable names that are explicit enough. `returnValue` could be `encryptedPassword`

Reply:
Nice. Thanks for that